automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/WorkspaceDataPage.scala

```diff
@@ -90,16 +90,22 @@ class ImportMetadataModal(implicit webDriver: WebDriver) extends OKCancelModal("
   private val fileUploadInput = FileSelector("data-upload-input" inside this)
   private val confirmUploadMetadataButton = Button("confirm-upload-metadata-button" inside this)
   private val uploadSuccessMessage = Label("upload-success-message" inside this)
+  private val chooseFileButton = Button("choose-file-button" inside this)
 
   /**
     * Imports metadata from a file.
     */
   def importFile(file: String): Unit = {
     importFromFileButton.doClick()
     fileUploadInput.selectFile(file)
+
     confirmUploadMetadataButton.awaitEnabledState()
     confirmUploadMetadataButton.doClick()
+    confirmUploadMetadataButton.awaitNotVisible()
```

Comment:
defensive code to avoid failure because Modal window don't change, only contents change.